db/migrate/20170723174039_create_workshops.rb

```diff
@@ -7,6 +7,7 @@ def change
       t.text :venue_address
       t.string :google_maps_url
       t.time :start_time
+      t.string :alternative_date
```

Comment:
Hi @DanielaValero 

Thanks for putting all of this together! 

This migration has already been run in the production environment and created the `workshops` table, and we can't run it again.

This change would need to go in a new migration, which can be done by running something like `rails generate migration <migration name>`. Then when we deploy the new migration can be run against the production environment and it's all good.

Let me know if this makes sense or not?

Reply:
Good point! 

The output of the migration would look like this:

```
class AddAlternativeDateToWorkshops < ActiveRecord::Migration[5.1]
  def change
    add_column :workshops, :alternative_date, :string
  end
end
```

is that correct?

Reply:
That looks great!

Reply:
Awesome, thank you! I will update the PR

Reply:
Done! Ready for a second round of review